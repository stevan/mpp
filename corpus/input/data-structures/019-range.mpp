my $range = 1..10;