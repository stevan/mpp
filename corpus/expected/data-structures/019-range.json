[
  {
    "type": "Declaration",
    "declarator": "my",
    "variable": {
      "type": "Variable",
      "name": "$range"
    },
    "initializer": {
      "type": "BinaryOp",
      "operator": "..",
      "left": {
        "type": "Number",
        "value": "1"
      },
      "right": {
        "type": "Number",
        "value": "10"
      }
    }
  }
]